src/Bonnier/WP/WypeResetpassword/Services/BmdValidateLoginService.php

```diff
@@ -28,6 +28,6 @@ public function validateSubscription($subscriptionNumber, $local)
 
         $responseBody = json_decode($response->getBody());
 
-        return $responseBody->IsValid;
+        return ['IsValid' => $responseBody->IsValid, 'Prefix' => $responseBody->Prefix];
```

Comment:
hvorfor ikke bare returnere responseBody i setedet for at lave en ny array med præcis samme keys ? :)